.ruby-version

```diff
@@ -1 +1 @@
-2.4
+2.4.10
```

Comment:
rbenc just refused to switch Ruby version without specifying minor release. Just let me know if you prefer that I submit this in a separate PR